### PR TITLE
*: support case sensitive in binlog-filter. column-mapping. table route

### PR DESCRIPTION
**Pull request by IANTHEREAL**
@csuzhangxc PTAL

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@GregoryIan after this PR merged, maybe we need to update some case convert functions like `ToLower` in *tidb-enterprise-tools* ?

File: pkg/binlog-filter/filter_test.go [hunk 1]
```diff
@@ -124,3 +124,45 @@ func (t *testRouterSuite) TestRoute(c *C) {
 	_, err = filter.Filter("test_1_a", "abc", InsertEvent, "")
 	c.Assert(err, NotNil)
 }
+
+func (t *testRouterSuite) TestCaseSensitivetC(c *C) {
```
Comment (csuzhangxc) [inline pkg/binlog-filter/filter_test.go]: s/TestCaseSensitivetC/TestCaseSensitive?

File: pkg/binlog-filter/filter_test.go [hunk 2]
```diff
@@ -27,9 +27,9 @@ var _ = Suite(&testRouterSuite{})
 
 type testRouterSuite struct{}
 
-func (t *testRouterSuite) TestRoute(c *C) {
+func (t *testRouterSuite) TestFilter(c *C) {
```
Comment (csuzhangxc) [inline pkg/binlog-filter/filter_test.go]: s/testRouterSuite/testFilterSuite ?

File: pkg/binlog-filter/filter.go [hunk 3]
```diff
@@ -92,6 +92,12 @@ type BinlogEventRule struct {
 	Action ActionType `json:"action" toml:"action" yaml:"action"`
 }
 
+// ToLower covert schema/table parttern to lower case
```
Comment (csuzhangxc) [inline pkg/binlog-filter/filter.go]: you can update typo in L71.  `ClassifyEevent` to `ClassifyEvent`

Comment (csuzhangxc) [reply pkg/binlog-filter/filter.go]: s/parttern/pattern

File: pkg/column-mapping/column.go [hunk 4]
```diff
@@ -242,7 +261,12 @@ func (m *Mapping) HandleDDL(schema, table string, columns []string, statement st
 		return statement, nil, nil
 	}
 
-	info, err := m.queryColumnInfo(schema, table, columns)
+	schemaL, tableL := schema, table
```
Comment (csuzhangxc) [inline pkg/column-mapping/column.go:269]: also convert case sensitive  for `schema`, `table` in `HandleDDL`?

**Comment by csuzhangxc** [comment]
rest LGTM

**Comment by IANTHEREAL** [comment]
yep, we maybe use it to support case sensitive

**Comment by IANTHEREAL** [comment]
@csuzhangxc PTAL

**Review by csuzhangxc — APPROVED** [review]
LGTM

**Pull request merged**